components/search/SearchForm.vue

```diff
@@ -3,32 +3,54 @@
     inline
     @submit.prevent="submitForm"
   >
-    <b-form-input
-      v-model="query"
-      :aria-label="$t('search')"
-      :placeholder="$t('searchPlaceholder')"
-      name="query"
-      data-qa="search box"
-      @input="$emit('input', $event)"
-    />
-    <b-button
-      type="submit"
-      data-qa="search button"
-      variant="primary"
-    >
-      <span>
-        {{ $t('search') }}
-      </span>
-      <img
-        src="../../assets/img/magnifier.svg"
-        :alt="$t('search')"
+    <b-input-group>
+      <template
+        v-if="pillLabel"
+        v-slot:prepend
       >
-    </b-button>
+        <SearchBarPill
+          :text="pillLabel"
+          :remove-link-label="$t('removeEntityFilter', { entityLabel: pillLabel })"
```

Comment:
This should be supplied from the store too. The search form component should not need to know anything about entities.

Reply:
On second thoughts, all that should be needed is to rename the translation and placeholder in lang/en.js as its content is entity-agnostic anyway.

So change it in lang/en.js to: `"removeFilter": "Remove {filterLabel} filter"`